SUMMARY.md

```diff
@@ -80,7 +81,17 @@
   
   * [3.0.4](release-notes/ugandaemr-304.md)
   
-  * [3.1.0](release-notes/ugandaemr-310.md)
+  * [3.1.0](release-notes/ugandaemr-310.md) 
+ 
+* UgandaEMR Mobile
```

Comment:
@solemabrothers move the section above the release notes which should be the last

Reply:
Done